leg_fusion:
    filter_type: "EKF"                                          # The Kalman filter type: EKF = Extended Kalman Filter, UKF = Uncented Kalman Filter
    prune_named: true                                           # make sure identity tracks are unique (prunes the one with high stdev)
    cv_noise_params:                                            # The noise for the constant velocity prediction model
        x: 1.5
        y: 1.5
    std_limit: 1.5                                              # pruning tracks that have std(X) > std_limit
    detectors:                                                  # Add detectors under this namespace
        laser_tracker:                                          # Name of detector (used internally to identify them). Has to be unique.
            topic: "/laser_tracked_people_pose_array"           # The topic on which detections are published
            noise_params:                                       # The noise for the cartesian observation model
                x: 0.3
                y: 0.3
            observation_model: "CARTESIAN"                      # CARTESIAN observation model (POLAR is alternative)
            seq_size: 5                                         # minimum `seq_size` number of detections received in `seq_time` seconds
            seq_time: 0.5                                       #   to create new track
            matching_algorithm: "NN"                            # The algorthim to match different detections. NN = Nearest Neighbour, NN_LABELED for identifying detections, NNJPDA = NN Joint Probability Data Association
        radar_tracker:
            topic: "/radar_tracked_people_pose_array"
            noise_params:
                x: 0.2
                y: 0.2
            observation_model: "CARTESIAN"
            seq_size: 5
            seq_time: 1.5
            matching_algorithm: "NN"


